### ISSUE TITLE
Setup Issue with linux

### ISSUE THREAD

**Issue by abhij89** [4jean/lav_sms#49]
I don't see any requirements doc for this package. I tried installing this on windows with node v16.14, and nvm v7.17. Worked fine. I tried the same thing on ubuntu 20.04. It failed. Tried various versions of npm ranging from 7.0 to 8.7. It didn't work. This is the error I am getting. Any ideas to fix it?

![image](https://user-images.githubusercontent.com/15982920/165400777-18e2c0e4-064c-4219-9858-8feba2749eee.png)

**Comment by yungifez** [comment]
What command did you run

**Comment by yungifez** [comment]
I advice that if you aren't about to actively contribute to this project, that you leave npm alone 

The project already cones with the scripts and the css 

If not run npm update instead 

See

https://stackoverflow.com/questions/59354180/error-please-upgrade-your-dependencies-to-the-actual-version-of-core-js3

**Comment by abhij89** [comment]
it's required. you need to install node modules. 
I ran the standard command: `npm install` that's all. @yungifez

**Comment by yungifez** [comment]
He already has the css and js compiled

But since you insist run 

`npm update`

**Comment by abhij89** [comment]
yeah, I would still like to run it. and it results in the same error. 
I still don't understand why is it erroring out in one environment while working fine in the other.

![image](https://user-images.githubusercontent.com/15982920/165563202-c28df62a-45d7-40cf-853d-21892da49158.png)

**Comment by yungifez** [comment]
Since that didn't work

Run npm install --save core-js@^3

**Comment by abhij89** [comment]
Tried still the same but I don't think core-js is the issue, it has something to do with the node-sass. I tried installing node-sass directly but it still resulted in an error..not sure what's wrong with this package.

**Comment by yungifez** [comment]
I'll try recreating your issue

**Comment by abhij89** [comment]
any luck?

**Comment by yungifez** [comment]
I'm sorry at the moment I'm in school and have limited access to my computer

**Comment by yungifez** [comment]
I have seen questions on stack overflow with the suggestion to 

Clear cache first 
npm cache clean --force

Does this issue occur on just this project?

**Comment by abhij89** [comment]
yes, just this project. no problem.

**Comment by yungifez** [comment]
hello i'm back from school would you still like assistance
